packages/contracts/contracts/Unirep.sol

```diff
@@ -112,6 +116,7 @@ contract Unirep is IUnirep, VerifySignature {
                 x,
                 initialData[x]
             );
+            attestationCounter++;
```

Comment:
No need to increment the counter here. During signup any custom data must use nonce 0. This is enforced on line 110, we check that all the upper bits are 0.